app/controllers/idv_controller.rb

```diff
@@ -13,12 +14,8 @@ def index
       verify_identity
     elsif active_profile?
       redirect_to idv_activated_url
-    elsif idv_attempter_throttled?
-      irs_attempts_api_tracker.idv_verification_rate_limited(throttle_context: 'single-session')
-      analytics.throttler_rate_limit_triggered(
-        throttle_type: :idv_resolution,
-      )
-      redirect_to idv_session_errors_failure_url
+    elsif check_rate_limited_and_redirect
+      # do nothing
```

Comment:
This is a little awkward, right? We aren't exactly "doing nothing" here. We are leaning on the side effect of what we did in the condition